nxdrive/client/uploader/direct_transfer.py

```diff
@@ -86,6 +89,10 @@ def upload(
                 remote_parent_path=doc_pair.remote_parent_path,
                 remote_parent_ref=doc_pair.remote_parent_ref,
                 doc_pair=doc_pair.id,
+                headers={
+                    "NX-metric-directTransfer.duplicate.behavior": doc_pair.duplicate_behavior,
```

Comment:
Now that I read the implementation, we may need to specifiy that we are talking about files:
```
NX-metric-directTransfer.duplicate.file.behavior
```